libraries/Microsoft.Bot.Builder.AI.LUIS/Generator/InstanceData.cs

```diff
@@ -71,6 +71,8 @@ public class InstanceData
         /// Any extra properties.
         /// </value>
         [JsonExtensionData(ReadData = true, WriteData = true)]
+#pragma warning disable CA2227 // Collection properties should be read only (we can't change this without breaking binary compat)
```

Comment:
>CA2227 [](start = 24, length = 6)

I don't agree with these (they are a number of places).  We want to be able to modify these collections as well as replace them.  We need both from an API and serialization standpoint.

Reply:
see other comment.